community/case-studies/wayfair/000_index.md

```diff
@@ -0,0 +1,184 @@
+# Wayfair
+
+![wayfair](./media/wayfair.png)
+
+Development on tremor started in the autumn of 2018 as a point solution
+for introducing a configurable traffic shaping mechanism for our
+infrastructure platform engineering organization.
+
+There were two use cases being considered at the time - this use case
+had the highest priority. The other was distributed data replication for
+our Kafka-based environments.
+
+## The Origin of Tremor
+
+The [Traffic Shaping](./traffic-shaping)
+use case required that log information streaming from various systems be
+categorized and classified in-flight, so that temporally bound rate
+limits could be applied if, or rather when, our production estate has a
+working set that is in excess of our current capacity.
+
+As a 24x7x365 eCommerce environment with year on year higher peaks,
+higher volumetrics and higher loads on our systems - this is a frequent
+occurrence. Tremor was designed to remove the burden of scaling these
+systems from our system reliability and network operations center
+engineers.  
+  
+This use case saw the birth of tremor. It was inflexible - only
+supported connectivity kafka for ingres and elasticsearch downstream.
+And the classification and rate limiting logic was more or less
+hardwired.
+
+But, it was delivered in 6 weeks just in time for its first Cyber 5 (
+the long bank-holiday weekend in the US centered around Thanksgiving
+where US folk go peak shopping! ) - and it worked flawlessly.
+
+Although designed with our logging systems in mind, our metrics systems
+team saw that the same system would work for their environment (
+InfluxDB downstream rather than ElasticSearch, but the logic was
+basically the same ).
+
+Tremor was designed to detect or sense when downstream systems are
+suffering back-pressure and to intelligently - based on user defined
+logic - to react by adaptively tuning the data distribution to
```

Comment:
There's an extra "to".
... and to intelligently- based on user defined
logic- react